test/performance/io/format_fasta_benchmark.cpp

```diff
@@ -42,6 +44,12 @@ void write3(benchmark::State & state)
     {
         format.write(ostream, options, seq, id, std::ignore);
     }
+
+    ostream = std::ostringstream{};
+    format.write(ostream, options, seq, id, std::ignore);
+    size_t bytes_per_run = ostream.str().size();
+    state.counters["bytes_per_run"] = bytes_per_run;
```

Comment:
I'd change three lines to two in each of the four benchmarks:
```diff
- size_t bytes_per_run = ostream.str().size();
- state.counters["bytes_per_run"] = bytes_per_run;
- state.counters["bytes_per_second"] = bytes_per_second(bytes_per_run);
+ state.counters["bytes_per_run"] = ostream.str().size();
+ state.counters["bytes_per_second"] = bytes_per_second(state.counters["bytes_per_run"]);
```

Reply:
I don't know if this is really relevant.